### ISSUE TITLE
Rendering objects as semi-transparent.

### ISSUE THREAD

**Issue by danieljfarrell** [rdeits/meshcat-python#35]
It is possible to render objects as semi-transparent so that the interior remains visible? Or another alternative could be wireframing objects. I can see the different material classes in `geometry.py` such as `MeshPhongMaterial` and `MeshToonMaterial` but these shaders make the surface opaque. Maybe a surface texture is the way to do this? But I'm unsure how to proceed.

**Comment by matthieuvigne** [comment]
Hi @danieljfarrell,

I'm also interested in such a feature. As far as I can see, meshcat-python doesn't provide such an API. However, Three.js supports transparent objects, so it can be added quite easily. This 
[patch](https://github.com/rdeits/meshcat-python/files/2897776/transparency.patch.txt) adds a ```transparency``` and ```opacity``` field to the MeshMaterial object for configuring these properties.
So the following code

```
import meshcat

visualizer = meshcat.visualizer.Visualizer()
sphere = meshcat.geometry.Sphere(1.0)
material = meshcat.geometry.MeshLambertMaterial()
material.transparency = True
material.opacity = 0.5
visualizer["sphere"].set_object(sphere, material)
```

renders a transparent sphere:
![screenshot from 2019-02-24 11-06-55](https://user-images.githubusercontent.com/16670497/53297838-a0d80300-3824-11e9-860b-fcb76ff38b8d.png)

@rdeits Do you confirm this isn't currently possible with meshcat? If so, I can make a PR with this patch.

**Comment by danieljfarrell** [comment]
Would be really interested in knowing how you solved this problem. Would you upload your fork to GitHub?

**Comment by matthieuvigne** [comment]
@danieljfarrell Sure! There isn't anything more than what was in the patch, it's just a few lines of code to have meshcat communicate an extra attribute to Three.js 
FYI, I found the information about transparency in Three.js in the doc: https://threejs.org/docs/#api/en/materials/Material.opacity

I just created a PR for this issue.

**Comment by danieljfarrell** [comment]
Oh sorry! Missed the link to the patch in the original comment.

**Comment by rdeits** [comment]
Ok, this should now be available in meshcat v0.0.15. You should be able to `pip install --upgrade meshcat` to get the newest version. Thank you @matthieuvigne for the PR!
